stardog/admin.py

```diff
@@ -188,29 +195,26 @@ def restore(self, from_path, *, name=None, force=False):
 
         self.client.put('/admin/restore', params=params)
 
-    #TODO:
-    # def backup_all(self):
-    #     """
-    #     Create a backup of all databases on the server
-    #     https://stardog-union.github.io/http-docs/#operation/backupAll
-    #     :return:
-    #     """
+    def backup_all(self, location=None):
+        """
+        Create a backup of all databases on the server
+        https://stardog-union.github.io/http-docs/#operation/backupAll
+        :return:
+        """
+        if location:
+            self.client.put('/admin/databases/backup_all?to=' + location )
+        else:
+            self.client.put('/admin/databases/backup_all')
 
-    #TODO
-    # def restore_all(self):
-    #     """
-    #     Restore multiple databases from backup
-    #     https://stardog-union.github.io/http-docs/#operation/restoreAll
-    #     :return:
-    #     """
 
-    #TODO
-    # def get_all_metadata_options(self):
-    #     """
-    #     Get information on all database metadata properties, including description and example values
-    #     https://stardog-union.github.io/http-docs/#operation/getAllMetaProperties
-    #     :return:
-    #     """
+    def get_all_metadata_properties(self):
+        """
+        Get information on all database metadata properties, including description and example values
+        https://stardog-union.github.io/http-docs/#operation/getAllMetaProperties
+        :return:
+        """
+        self.client.get('/admin/config_properties')
```

Comment:
You should add a test for this to make sure something comes back. First you'll need to return the results:

```suggestion
        return self.client.get('/admin/config_properties')
```